docs/demos/array/gallery.js

```diff
@@ -42,7 +42,7 @@ export default class GalleryDemo extends React.Component<ArrayTypes<any>> {
           <Gallery
             value={value}
             refId={new RefId("gallery")}
-            imageServiceConfig={new ImgurService({
+            imageStorage={new ImgurStorage({
```

Comment:
@chihsuan, I prefer to pass a `serviceConfig` here
```js
serviceConfig={{
  customRequest: ({..}) => {...}
}}
```

Reply:
@abz53378: I've updated the components. But should this be like the image component that is written in the `packages/antd-array-gallery/src/index.js` or you prefer to pass serviceConfig here?